about.html

```diff
@@ -0,0 +1,134 @@
+                Web Convention are a gathering centered on web developers, mobile application developers, software engineers, CEO's of tech companies.
+                Alongside tech’s leaders and innovators, we’ve gathered authors, athletes, actors and more to talk about how tech touches their industries. 
+                Our world-class speakers will take to the stage to share their expertise through inspiring talks, interactive Q&As, and masterclasses covering more than 30 topics.
+
+
+                
+            </p>
+            <h2 id="contact">Please contact as by e-mail if you have any other questions about the event</h2>
+            <p id="email">contact@websummit.com</p>
+        </section>
+
+        <!-- Program Section -->
+        <section id="logo-details">
+            <h3>Web Summit 2022 logo</h3>
+            <p id="logo-info">The logo of WS (Web Summit) was created by Manel Hammouche on April, 2022 using "logopit" (a mobile application)
+            </p>
+            <img id="logo-image" src="images/logo.png" alt="">
+        </section>
+        
+        <div id="underline"></div>
+
+        <!-- Conventions Section -->
+        <section id="convecation">
+            <h3>See the past Web Summits</h3>
+            <p id="convecation-info">Take a look at the last two Web Summits which took place in Tokyo and NewYork
+            </p>
+            <div id="prev-coventions">
+                <div class="prev-covention">
+                    <img class="convecation-image" src="images/prev-conference1.jpg" alt="">
+                    <div class="centered">
+                        <h4>2019</h4>
+                        <p>WS 2018 in New York</p>    
+                    </div>
+                </div>
+                <div class="prev-covention">
+                    <img class="convecation-image" src="images/prev-conference2.jpg" alt="">
+                    <div class="centered">
+                        <h4>2020</h4>
+                        <p>WS 2020 in Tokyo</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- Sponsors Section -->
+        <section id="sponsors-section">
+            <h3>Sponsors</h3>
+            <div class="section-underline"></div>
+            <div id="sponsors">
+                <div class="sponsor">
+                    <img src="images/ps-logo.png" alt="Play Station">
+                </div>
+                <div class="sponsor">
+                    <img src="images/riot-logo.png" alt="Riot Logo">
+                </div>
+                <div class="sponsor">
+                    <img src="images/sony-logo.png" alt="Play Station">
+                </div>
+                <div class="sponsor">
+                    <img src="images/xbox-logo.png" alt="Play Station">
+                </div>
+                <div class="sponsor">
+                    <img src="images/nintendo-logo.png" alt="Play Station">
```

Comment:
- [ ] Please kindly remove the sponsor's section as it is not specified in the 
original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/170722660-8bb1722c-4f7e-48ae-b185-970164c39ea0.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/170722422-b9280169-3e7d-4bad-84fd-c3bbefe25158.png)